nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -996,7 +996,7 @@ in {
             ln -sf ${overrideConfig} ${datadir}/config/override.config.php
 
             # Do not install if already installed
-            if [[ ! -e ${datadir}/config/config.php ]]; then
+            if [[ ! -s ${datadir}/config/config.php ]]; then
```

Comment:
Perhaps it's better to fail loudly in that case:

* if it's a fresh install with `config.php` existing, but being empty, that's clearly a user-error.
* if the file gets truncated for whatever reason on an existing install, the re-installing will cause a failure later on or even worse corrupt the installation even more. Also, I'm not sure if a config.php will be written even if the install fails, but in that case it's harder to trace the cause of the error (i.e. that the file _was_ truncated before).

Am I missing a case where this would be a better option?

I.e.
* if it exists and is empty -> exit 1
* if it doesn't exist -> install